examples/vue-app/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { createApp } from 'vue'
+import './style.css'
+import App from './app.vue'
```

Comment:
`import App from './app.vue'` should `import App from './App.vue'`

Reply:
Should be good now, as well with the styles